package.json

```diff
@@ -73,5 +73,10 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/wagmi-dev/wagmi.git"
+  },
+  "pnpm": {
+    "overrides": {
+      "@wagmi/core": "~0.7.5"
```

Comment:
What is this for? (This https://github.com/family/connectkit/issues/42#issuecomment-1311221683?)

Reply:
Yeah, seems that the web3modal template is pulling in the wrong version of `@wagmi/core` since ConnectKit isn't compatible w/ this version yet.